rafthttp/transport.go

```diff
@@ -110,10 +116,26 @@ type Transport struct {
 	prober probing.Prober
 }
 
-func (t *Transport) Start() {
+func (t *Transport) Start() error {
+	var err error
+	// stream uses long term connection, and maintains heartbeat on the connection.
+	// It expects underlying connection returns error when taking long time to read and write,
```

Comment:
Please directly explain why do we want to set read/write timeout per read/write without going through all these circles. The comment is not directly enough.